deployments/BridgeDeps.Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Image with dependencies required to build projects from the bridge repo.
+#
+# This image is meant to be used as a building block when building images for
+# the various components in the bridge repo, such as nodes and relayers.
+FROM ubuntu:xenial
```

Comment:
Any particular reason why to use `xenial` (16.04 LTS) and not `focal` (20.04 LTS)? `xenial` is pretty outdated.

Reply:
I guess it was simply just moved, but let's maybe do an upgrade since there is a good occasion :)

Reply:
I've tried switching the base image to `focal`, but I'm having issues getting the Docker images built. For some reason Docker can't find some files 🤷 

```
Step 18/31 : RUN cargo build --release --verbose -p ${PROJECT}
 ---> Running in eba424f7d692
error: failed to read `/parity-bridges-common/bin/millau/node/Cargo.toml`

Caused by:
  No such file or directory (os error 2)
The command '/bin/sh -c cargo build --release --verbose -p ${PROJECT}' returned a non-zero code: 101
```

If you're curious, the steps to reproduce are:
- `s/xenial/focal/g`
- Add `ENV DEBIAN_FRONTEND=noninteractive` to the top of `BridgeDeps.Dockerfile` (prevents `tzdata` from stalling the build)
- Build deps
- Build relay or node

I would say that unless there is a strong reason _for_ switching, we should keep the `xenial` base images and log an issue to upgrade them at a later date.

Reply:
We still have until April 2021. That is when standard support for `xenial` ends ;-)

Reply:
logged here: https://github.com/paritytech/parity-bridges-common/issues/626